src/Annotation/Enumerator.php

```diff
@@ -6,70 +6,48 @@
 
 namespace Hostnet\Component\AccessorGenerator\Annotation;
 
-use Doctrine\Common\Annotations\Annotation\Enum;
-
 /**
  * @Annotation(target={"ANNOTATION", "PROPERTY"})
  */
 class Enumerator
 {
     /**
      * References the Enum class for the parameter collection.
-     *
-     * @var string
```

Comment:
`doctrine/annotations` actually uses these variables in their validation, please keep these intact

https://www.doctrine-project.org/projects/doctrine-annotations/en/latest/custom.html

Reply:
You are correct, they should be kept. Will be fixed in the next commit I push. :+1: